wsd/DocumentBroker.cpp

```diff
@@ -1802,6 +1808,38 @@ bool DocumentBroker::autoSave(const bool force, const bool dontSaveIfUnmodified)
     return sent;
 }
 
+void DocumentBroker::autoSaveAndStop(const std::string& reason)
+{
+    // Don't hammer on saving.
+    if (_saveManager.timeSinceLastSaveRequest() >= std::chrono::seconds(1) &&
```

Comment:
Anytime I see a hard-coded time I'm un-happy =) particularly when it's in seconds.
Does this delay document close / cleanup / re-load / shutdown ? why do we need to wait ? what essential state changes here ?
Still perhaps better than the 5 seconds it replaced - but ... que? =)

Reply:
It's a small delay to avoid hammering Core with `.uno:Save` requests, which causes high cpu, floods the queues and doesn't allow for Core to get to idle. It's an arbitrary delay to avoid the above (which was observed in the wild) while being short enough as to not hold up unloading.

No queues here, in this 'unloading' state this function will be invoked on each poll, so it can really overheat quickly. Here we just skip issuing `.uno:Save` for those seconds since the last request/response.

Happy to move these compile-time constants somewhere central and name them, of course, but they get a bit detached if used only once, such as this case.